Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -167,17 +167,46 @@ protected override void Dispose(bool disposing)
 			{
 				_disposed = true;
 
-				if (_titleViewRenderer != null)
+				Device.Info.PropertyChanged -= DeviceInfoPropertyChanged;
+			
+				if (NavigationPageController != null)
 				{
-					Android.Platform.ClearRenderer(_titleViewRenderer.View);
-					_titleViewRenderer.Dispose();
-					_titleViewRenderer = null;
+					var navController = NavigationPageController;
+
+					navController.PushRequested -= OnPushed;
+					navController.PopRequested -= OnPopped;
+					navController.PopToRootRequested -= OnPoppedToRoot;
+					navController.InsertPageBeforeRequested -= OnInsertPageBeforeRequested;
+					navController.RemovePageRequested -= OnRemovePageRequested;
 				}
 
+				if (Current != null)
+				{
+					Current.PropertyChanged -= CurrentOnPropertyChanged;
+				}
+
+				FragmentManager fm = FragmentManager;
```

Comment:
Just making sure you deleted the API version line (`if ((int)Build.VERSION.SdkInt >= 17)`) from below on purpose. Because we target higher Android version only now, right?

Reply:
Yes it's on purpose.
I deleted it because :
- Everywhere in the application where FragmentManager we don't have this
- I checked the documentation and nothing in this bloc is limited to api 17 (we use the support library here)